docs/40_node-operation/10_getting-started/60_logging/20_third_party_logging/10_deep_mind_logger.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Deep-mind Logger Integration
+---
+
+## Overview
+
+The `Deep-mind logger` is part of the `dfuse` [platform]([https://dfuse.io/](https://dfuse.io/)) which is a highly scalable and performant [open-source]([https://github.com/dfuse-io/dfuse-eosio/tree/master](https://github.com/dfuse-io/dfuse-eosio/tree/master)) platform for searching and processing blockchain data.
```

Comment:
Linking to https://dfuse.io/ is problematic. 

The project no longer exists at that domain, and streamingfast which it links to is not synonymous tech.

Reply:
I have a new link, I just forgot to submit it.